graphql/schema/types/project.graphql

```diff
@@ -177,6 +183,7 @@ Project models single repository on GitHub.
 type Project {
   id: String!
   admins: [String] @requireProjectFieldAccess
+  banner: ProjectBanner
```

Comment:
Something maybe up with your branch history.

Reply:
Ah no, I just reordered these fields because they were not in alphabetical order